.goreleaser.yml

```diff
@@ -0,0 +1,15 @@
+archive:
+    wrap_in_directory: true
+    replacements:
+        darwin: Darwin
+        linux: Linux
+        windows: Windows
```

Comment:
I didn't actually test on Windows, bad me. I'll do that.

Reply:
I wasn't actually building the windows binary, oops. I've fixed that, but I'm not sure of the best way to test on windows. I could setup a VM, but that's a somewhat annoying process. I could email the binary to my personal account, but I'm not sure that won't violate a security policy of some sort. A third option, and what I'm leaning towards, is to wait until after a release and then test it on my personal machine. I think there is a high likelihood that it will `just work`, and a very low likelihood that any real users will run across it before I have a chance to test it.